invenio_audit_logs/resources/resource.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Logs resource."""
+
+from flask import g
+from flask_resources import Resource, resource_requestctx, response_handler, route
+from invenio_records_resources.resources.records.resource import (
+    request_extra_args,
+    request_search_args,
+    request_view_args,
+)
+from invenio_records_resources.resources.records.utils import search_preference
+
+
+#
+# Resource
+#
+class AuditLogsResource(Resource):
+    """Resource layer for audit-logs."""
+
+    def __init__(self, config, service):
+        """Constructor."""
+        super(AuditLogsResource, self).__init__(config)
+        self.service = service
+
+    def create_blueprint(self, **options):
+        """Create the blueprint."""
+        # We avoid passing url_prefix to the blueprint because we need to
+        # install URLs under both /audit-logs and /api/audit-logs. Instead we
+        # add the prefix manually to each route (which is anyway what Flask
+        # does in the end)
```

Comment:
I am not sure about this one... did you check how the other modules do the distinction between API and UI urls ?

Reply:
We don't need a distinction here because we don't have a UI view, other modules follow the same in this case.